instabug/README.md

```diff
@@ -0,0 +1,34 @@
+# Instabug integration
+## Overview
+
+[Instabug][1]'s comprehensive platform empowers developers to monitor, prioritize, and debug performance and stability issues throughout the entire mobile app development lifecycle. Over 15,000 companies utilize the unmatched performance monitoring and crash reporting features, allowing their team of developers to leverage critical data in real time. Verizon, PayPal, eBay, and more of today's top-rated apps rely on our automated reports, alerts, and data solutions to boost their mobile app performance, prepare for smoother release cycles, and enable further developments and scalability.
```

Comment:
We try to avoid marketing copy in the overview section. The overview should include:

1. What does instabug do in 1-2 sentences
2. What will customers get from the integration
3. What data is sent in